modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator does the following:
+
+// seed cluster, some things are cluster specific that are not included
+// all operator, LCAO, OADP, LVMS, included in the seed image, telco day2 stuff is incl
```

Comment:
We will need to expand on this

Reply:
@browsell Could you please point me in the right direction on what would you like to see here exactly?